### PR TITLE
Tensorflow Training Sample

### PR DESCRIPTION
**Pull request by raymondlo84**
Added the Tensorflow Training Sample (based on COLAB training tutorial)

### CODE REVIEW

**Comment by ryanloney** [comment]
Let's update to include the training part with data augmentation inside the notebook, since it runs on CPU. We can still link to Colab, but let's show the end to end flow here too. I am approving and Helena can merge to develop.

**Review by ryanloney — APPROVED** [review]

**Comment by raymondlo84** [comment]
@helena-intel @ryanloney I combined the tutorial into this and now it's a bigger one page with everything. So if you click Run->All it will work! This is pretty cool indeed.

**Review by helena-intel — CHANGES_REQUESTED** [review]
Can you add the License back to the notebook? Apache License requires that. https://www.apache.org/licenses/LICENSE-2.0

Also, for the parts that are commented out, can you either remove them or add a bit of text about why they are commented out and why a user may want to uncomment it?

File: notebooks/103-tensorflow-training-openvino/103-tensorflow-training-openvino.ipynb [hunk 1]
```diff
@@ -0,0 +1,1141 @@
...
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Run the Inference on the Input image...\n",
+    "inp_img_url = \"https://upload.wikimedia.org/wikipedia/commons/4/48/A_Close_Up_Photo_of_a_Dandelion.jpg\"\n",
+    "inp_file_name = 'A_Close_Up_Photo_of_a_Dandelion.jpg'\n",
+    "\n",
+    "#Download the image from the storage\n",
+    "output = subprocess.check_output(f\"curl {inp_img_url} --output {inp_file_name}\", shell=True)\n",
```
Comment (helena-intel) [inline notebooks/103-tensorflow-training-openvino/103-tensorflow-training-openvino.ipynb:1080]: Is curl included on a default Ubuntu 18.04 desktop install? http://releases.ubuntu.com/18.04/ubuntu-18.04.5-desktop-amd64.manifest seems to indicate it isn't. It's safer to use urllib.request.urlretrieve. I can also change this after merge if you want.

File: notebooks/103-tensorflow-training-openvino/103-tensorflow-training-openvino.ipynb [hunk 2]
```diff
@@ -0,0 +1,1141 @@
...
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "class_names=['daisy', 'dandelion', 'roses', 'sunflowers', 'tulips']\n",
+    "\n",
+    "model_xml = f'flower_ir.xml'\n",
```
Comment (helena-intel) [inline notebooks/103-tensorflow-training-openvino/103-tensorflow-training-openvino.ipynb:1045]: A bit above you define ir_path that refers to this file.

File: notebooks/103-tensorflow-training-openvino/103-tensorflow-training-openvino.ipynb (hunk 1 above)
Comment (raymondlo84) [reply notebooks/103-tensorflow-training-openvino/103-tensorflow-training-openvino.ipynb:1080]: I think we will do another update on that!

**Comment by raymondlo84** [comment]
> Can you add the License back to the notebook? Apache License requires that. https://www.apache.org/licenses/LICENSE-2.0
> 
> Also, for the parts that are commented out, can you either remove them or add a bit of text about why they are commented out and why a user may want to uncomment it?

Done and done! =)

**Comment by helena-intel** [comment]
Thanks Raymond! I'll merge to develop.

**Review by helena-intel — APPROVED** [review]

**Pull request merged**